File: 2TviAmenamecBajanarary/2tiv.js
<script>
function maxComDen(a,b){
    let max,min
    
    if (a>b){
        max = a
        min = b
    }
    else {
        min = a 
        max = b 
    }
    
    for (let i=min; i>0; i--) {
        let d = min%i
        if (d==0) {
            d=max%i
            if (d==0) {
                return i 
                
            }
                
        }
    }
}
console.log(maxComDen(24,6));  
</script>

File: 2TviAmenamecBajanarary/2tiv.cpp
#include <iostream>

int main()
{
    int a,b;
    int max,min;
    std::cout<<"a "; 
    std::cin>>a;
    std::cout<<"b ";
    std::cin>>b;
    if (a>b){
        max = a;
        min = b;
        
    }
    else {
        max = b;
        min = a;
        
    }
    for (int i = min; i>0; i--){
        if (min%i==0 && max%i==0){
            std::cout<<"answer: " << i;
            return 0;
        }
    }
    return 0;
}
